apps/system_apps/systemctl/systemctl.py

```diff
@@ -1,29 +1,41 @@
-
-import subprocess
```

Comment:
Start, stop, restart, reload, enable and disable still rely on the subprocess module, though - so, they're temporarily broken =(

Reply:
Good catch - it should be fixed now.